flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -358,6 +358,17 @@
                                     .list(text("'ng': new generation scheduler"))
                                     .build());
 
+    @Documentation.Section({
+        Documentation.Sections.EXPERT_SCHEDULING,
+        Documentation.Sections.ALL_JOB_MANAGER
+    })
+    public static final ConfigOption<Integer> MIN_PARALLELISM_INCREASE =
+            key("jobmanager.declarative-scheduler.min-parallelism-increase")
+                    .intType()
+                    .defaultValue(1)
+                    .withDescription(
+                            "Configure the minimum increase in parallelism for a job to scale up.");
```

Comment:
I would expect that CI won't pass because the configuration documentation has not been updated.